src/status_im2/contexts/quo_preview/main.cljs

```diff
@@ -219,6 +220,10 @@
                        {:name      :title-input
                         :options   {:topBar {:visible true}}
                         :component title-input/preview-title-input}]
+   :keyboard          [{:name      :keyboard-key
```

Comment:
this key should match figma to make it easy for devs to know where it's corresponding to
i.e `:numbered-keyboard`

Reply:
Done